cheroot/ssl/pyopenssl.py

```diff
@@ -352,11 +352,20 @@ def wrap(self, sock):
     def _password_callback(
         self,
         password_max_length,
-        _verify_twice,
-        password,
+        verify_twice,
+        password_or_callback,
         /,
     ):
         """Pass a passphrase to password protected private key."""
+        if callable(password_or_callback):
+            password = password_or_callback()
+            if verify_twice and password != password_or_callback():
+                raise ValueError(
```

Comment:
@jatalahd could you add a test to cover this line?

https://app.codecov.io/gh/cherrypy/cheroot/commit/529dd97b88d902fe15931e79fd8fd93012f1fd77/blob/cheroot/ssl/pyopenssl.py?flags%5B0%5D=pytest&dropdown=coverage#L363